src/components/List.js

```diff
@@ -1,22 +1,37 @@
 import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
-import styles from '../List.module.css';
 import Details from './Details';
-import { updatePurchaseDate, deleteItem } from '../lib/firebase.js';
-import { withStyles } from '@material-ui/core/styles';
+import { updatePurchaseDate } from '../lib/firebase.js';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
 import { green, orange, red, grey } from '@material-ui/core/colors';
 import {
   FormControlLabel,
   Checkbox,
   IconButton,
   Grid,
   Paper,
+  TextField,
   Typography,
+  InputAdornment,
+  Box,
+  Dialog,
+  Button,
 } from '@material-ui/core';
 import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
 import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
 import CircleUnchecked from '@material-ui/icons/RadioButtonUnchecked';
 import CircleCheckedFilled from '@material-ui/icons/CheckCircle';
+import SearchIcon from '@material-ui/icons/Search';
+import { NoPaddingClearIcon } from './ClearIconButton.js';
+import { Helmet } from 'react-helmet';
+import DeleteModal from './DeleteModal';
+
+const useStyles = makeStyles({
+  box: {
+    height: '48vh',
```

Comment:
This style falls apart a little bit depending on how tall the browser window is:
![image](https://user-images.githubusercontent.com/10378331/94364850-ee1bf680-0091-11eb-9bc8-e1399dceeb27.png)